src/index.html

```diff
@@ -7,12 +8,62 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="container"
+      data-qa="card"
+    >
+      <div class="container__img">
+        <img
+          class="img"
+          src="images/comp.png"
+          alt="iMac"
+        />
+      </div>
+      <div class="container__title">
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="container__code">Product code: 195434</div>
+      <div class="container__stars">
+        <ul class="stars">
```

Comment:
Fix the tag and add the modifier class
```suggestion
        <div class="stars stars--4">
```

Reply:
Тег змінив, а модифікатор такий як був не можна лишати? 
<div class="stars">
          <div class="stars__star"></div>

Reply:
`<div class="stars">
          <div class="stars__star"></div>`